mesonbuild/machinefile.py

```diff
@@ -1,6 +1,8 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright 2013-2024 Contributors to the The Meson project
+# Copyright © 2024 Intel Corporation
 
+from __future__ import annotations
```

Comment:
Since I mentioned this regarding the other PR -- which also adds a `__future__` import -- by saying that it is needed in machinefile.py too, I would really prefer if we just added the missing imports to both files at the same time and the same commit. :)